### ISSUE TITLE
How should I modify the training params to achieve the accuracy in your paper?

### ISSUE THREAD

**Issue by cloudcjf** [kxhit/SG_PR#9]
Hi, I am retraining your model with default settings on a  GTX1080 GPU, except the batch size is set to 256, and the nodes used are from semantic Kitti presented by you. When I first test sequence 08, which means sequences 00-07,09,10 as the training sets, the F1 max should achieve 0.900 if I wasn't missing any point. However, the best training result is only around 0.66(F1 max).  Could you please give some suggestions on this situation? 
![image](https://user-images.githubusercontent.com/54716067/124448181-b9716400-ddb4-11eb-8bf3-a68a29f07708.png)

**Comment by kxhit** [comment]
I would recommend to check by the following steps:
1. Run eval_batch as described in the README.md.
2. Run eval_batch with your model to check the results.
3. Could you provide the training logs or the evaluation results during training? I wish to confirm if the training process goes well. Make sure the data augmentation (e.g. randomly rotate) is opened.
4. The param batch size should not affect too much but you can train again with batch size 128 and make sure you are testing with the best_model in the saved models.

Hope it helps and let me know if the results are improved.

**Comment by cloudcjf** [comment]
I rechecked the README.md and finally figured out the mistake I had made. I use the pair_list_dir '../SG_PR_DATA/eval/3_20' both for training and evaluation. What a fault! This time I retrained the model with 20 epochs, and the F1 max score increased to 0.8. I'm sure it will reach the ideal result as I go on training. Sorry for wasting your time. By the way, have you ever tested training the model only to use one-hot encoded labels without coordinates of points? In my opinion, GCN is a model that can classify different types of nodes. So I wonder how much contribution the coordinates can make. 
Thanks again for your patience and quick response!

**Comment by kxhit** [comment]
Glad you resolved the problem!
Well, I think geometric coordinates are essential for place recognition tasks as we human usually recognize a scene by the relations (both geometric and semantic relation) among semantic objects which is the main motivation of this work. I agree with that GCNs can aggregate multi-modal features but still need proper insightful designs. I suppose you can easily test your ideas with these code and I would be glad to know the results if you would like to share and we may have further discussions later. Wish you good luck!

**Comment by cloudcjf** [comment]
Sure I will share my results with you. Your work proves to be a very notable one and worth working on further. I wonder if we could have discussions via email. Since my question was resolved, I will close this issue first.

**Comment by kxhit** [comment]
Thanks! This work is a naive try towards semantic-level place recognition. All discussions are welcomed!
